WebCompiler/package.csproj

```diff
@@ -20,7 +20,7 @@
   <!--PackageId, Version-->
   <PropertyGroup>
     <PackageId>Excubo.WebCompiler</PackageId>
-    <VersionPrefix>3.1.3</VersionPrefix>
+    <VersionPrefix>3.2.2</VersionPrefix>
```

Comment:
Please undo this change. I'll release a new version right after merging with a new version number.